doc/api/stream.md

```diff
@@ -1159,6 +1159,31 @@ readable stream will release any internal resources.
 Implementors should not override this method, but instead implement
 [`readable._destroy`][readable-_destroy].
 
+##### readable[Symbol.asyncIterator]
```

Comment:
This should be `@@asyncIterator` to be consistent with existing methods in docs like https://nodejs.org/api/url.html#url_urlsearchparams_iterator.

Reply:
We should maybe make the doc tool automatically link `@@foo` to the corresponding MDN link?

Reply:
We *could*. On the other hand it's not much help for `@@asyncIterator` because it's not yet documented.